buildtest/defaults.py

```diff
@@ -31,11 +31,12 @@
 
 # dictionary used for storing status of builds
 BUILDTEST_BUILD_HISTORY = {}
-BUILDTEST_CONFIG_FILE = os.path.join(BUILDTEST_ROOT, "settings.yml")
-BUILDTEST_CONFIG_BACKUP_FILE = os.path.join(BUILDTEST_ROOT, "settings.yml.bak")
+BUILDTEST_CONFIG_FILE = os.path.join(BUILDTEST_ROOT, "settings.json")
+BUILDTEST_CONFIG_BACKUP_FILE = os.path.join(BUILDTEST_ROOT, "settings.json.bak")
 
 # TESTCONFIG_ROOT is the root directory where test configurations are found
 # configs can be specified as full paths or relative to this path
 TESTCONFIG_ROOT = os.path.join(BUILDTEST_ROOT, "site")
-DEFAULT_CONFIG_FILE = os.path.join(root, "settings.yml")
+DEFAULT_CONFIG_FILE = os.path.join(root, "settings.json")
+DEFAULT_CONFIG_SCHEMA = os.path.join(root, "config_schema.json")
```

Comment:
I don't think we should start storing schemas in the root. I'd suggest a subfolder under buildtest.

Reply:
I put the files under ``buildtest/settings/`` to imply this is for buildtest settings and named the two files ``settings.json`` and ``settings.schema.json``.